log4j2-ecs-layout/src/test/java/co/elastic/logging/log4j2/IntegrationTest.java

```diff
@@ -0,0 +1,35 @@
+package co.elastic.logging.log4j2;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.config.ConfigurationSource;
+import org.apache.logging.log4j.core.config.Configurator;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class IntegrationTest {
+
+    @BeforeEach
+    public void init() throws IOException {
+//        ConfigurationSource source = new ConfigurationSource(new FileInputStream("log4j2.properties"));
+        Configurator.initialize(null, "log4j2.properties");
+    }
+
+    @Test
+    public void testxxx(){
+        Logger logger = LogManager.getLogger(getClass());
```

Comment:
```suggestion
        Logger logger = ctx.getLogger(getClass().getName());
```

Reply:
but we don't have `ctx` variable available in classes. Standard way to create a logger is with `LogManager.getLogger`

Reply:
I think it's just an issue of the test setup. I'm not sure how to do it properly tbh but in a non-test setting, the context should be available via `LoggerContext.getContext()`. How does ES set up the logging?

Reply:
the configuration is done here https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/common/logging/LogConfigurator.java 
on startup, before the values like `cluster.uuid` are available (They are received on first cluster state update)
Loggers are always created with `LogManager.getLogger`